examples/sw/benchmarks/coremark/ibex/core_portme.c

```diff
@@ -0,0 +1,230 @@
+/*
+        File : core_portme.c
+*/
+/*
+        Author : Shay Gal-On, EEMBC
+        Legal : TODO!
+*/
```

Comment:
Uncertain if we should include the lowRISC license header on this too, it's a copy of a file from the coremark repository with some changes.

I note it doesn't have a lowRISC license header in the OT repository, this also applies to the other portme files

Reply:
Just put a normal lowRISC header on it.

Reply:
Please remove everything starting from the `// This file is ...`, and replace it with two lines  under the `Copyright lowRISC contributors.`

```
Copyright 2018 Embedded Microprocessor Benchmark Consortium (EEMBC). 
Original Author: Shay Gal-on
```

That's all information we need to maintain in a shorter form.